internal/generator/main_test.go

```diff
@@ -0,0 +1,96 @@
+package main
```

Comment:
```suggestion
package generator_test
```

and the file is `generator_test.go`